yt/yt/tests/integration/queues/test_consumer_registrations.py

```diff
@@ -817,6 +816,8 @@ def test_multicluster_symlink_registrations(self, create_registration_table):
 class TestDataApi(TestQueueConsumerApiBase, ReplicatedObjectBase, TestQueueAgentBase):
     NUM_TEST_PARTITIONS = 2
 
+    NUM_REMOTE_CLUSTERS = 2
```

Comment:
Let's make a separate suite for multi-cluster API tests. The multi-cluster set-up/tear-down takes a while and would make all the other tests in the suite longer.